ArduCopter/Copter.h

```diff
@@ -269,7 +269,7 @@ class Copter : public AP_Vehicle {
     class SurfaceTracking {
     public:
         // get desired climb rate (in cm/s) to achieve surface tracking
```

Comment:
we should change the comment above the "get_surface_offset" method to match it's new purpose in life (both in the .h and the .cpp)